ui/app/src/components/RemoteRepositoriesDiffDialog/RemoteRepositoriesDiffDialogUI.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2007-2021 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Resource } from '../../models/Resource';
+import { FileDiff } from '../../models/Repository';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+
+const tabsHeight = 450;
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    diffTab: {
+      height: tabsHeight,
+      overflowX: 'auto'
+    },
+    diffContent: {
+      fontSize: '14px',
+      background: 'none',
+      border: 'none'
+    },
+    splitView: {
+      height: Math.floor(tabsHeight / 2),
+      overflowX: 'auto'
+    }
+  })
+);
+
+export interface RemoteRepositoriesDiffDialogUIProps {
+  resource: Resource<FileDiff>;
+  tab: number;
+  handleTabChange(e: React.ChangeEvent<{}>, newValue: number): void;
+}
+
+export default function RemoteRepositoriesDiffDialogUI(props: RemoteRepositoriesDiffDialogUIProps) {
```

Comment:
export RemoteRepositoriesDiffDialogUI...

export default RemoteRepositoriesDiffDialogUI;

Change all to follow this pattern.